client/src/pages/Background.js

```diff
@@ -0,0 +1,306 @@
+    this.setState({ password: e.target.value });
+  };
+  signUp = () => {
+    this.props.history.push({
+      pathname: "/signup",
+      state: {},
+    });
+  };
+  handleClick = () => {
+//    this.validation();
+    this.setState({ openSnack: (this.state.experience === "" ) ? true : false });
+  };
+  async validation() {
+    //fetch here
+    const res = fetch("/signin", {
+      method: "post",
+      headers: { "Content-Type": "application/json" },
+      body: {
+        email: this.state.email,
+        password: this.state.password,
+      },
+    })
+      .then((response) => response.json())
+      .then((responseJson) => {
+        return responseJson.cookie;
+      })
+      .catch((error) => {
+        console.error(error);
+      });
+
+    var success = false;
+    if (true) {
+      success = true;
+    }
+    await this.setState({
+      severity: success ? "success" : "error",
+      message: success ? "Sign in successfully!" : "Invalid email or password!",
+    });
+    if (success === true) {
+    }
+  }
+  handleClose = (event, reason) => {
+    if (reason === "clickaway") {
+      return;
+    }
+    this.setState({ openSnack: false });
+  };
+  languageChange = (e) => {
+    this.setState({language: e.target.value});
+  };
+  experienceChange = (e) => {
+    this.setState({experience: e.target.value});
+  };
+
+  render() {
+    const { classes } = this.props;
+    return (
+      <div className={classes.root}>
+        <Paper className={classes.paper} elevation={0}>
+          <Grid container spacing={0}>
+            <Grid item>
+              <Grid className={classes.image}></Grid>
+            </Grid>
+            <Grid className={classes.text} item xs={12} sm container>
+              <Grid item xs container direction="column" spacing={0}>
+                <Grid item xs>
+                  <Typography
+                    className={classes.title}
+                    align="center"
+                    variant="h2"
+                  >
+                    Tell about your background
+                  </Typography>
+                  <Grid>
+                    <Typography className={classes.question}>Your language:</Typography>
+                    <FormControl
+                      variant="outlined"
+                      fullWidth
+                    >
+                      <Select
+                        labelId="demo-simple-select-outlined-label"
+                        id="demo-simple-select-outlined"
+                        value={this.state.language}
+                        onChange={this.selectChange}
+                      >
+                        <MenuItem value={"React"}>React</MenuItem>
+                        <MenuItem value={"JavaScript"}>JavaScript</MenuItem>
+                        <MenuItem value={"Node.js"}>Node.js</MenuItem>
+                        <MenuItem value={"Angular"}>Angular</MenuItem>
+                        <MenuItem value={"Python"}>Python</MenuItem>
```

Comment:
A lot of these can be made global variables in this file and then you can map through them to generate the MenuItems

Reply:
Use map instead of these lines